tests/ops/qubit/test_matrix_ops.py

```diff
@@ -518,7 +518,7 @@ def circuit(x):
             loss = circuit(x)
 
         grad = tape.gradient(loss, x)
-        expected = -tf.math.sin(x)
+        expected = -tf.math.sin(x)  # pylint: disable=invalid-unary-operand-type
```

Comment:
I'm confused by this change. Wasn't `test_matrix_ops.py` lint-compatible already?

Reply:
I believe my `pre-commit` complained about this because my local pylint version is newer. Let me make sure I install the correct pylint version.

We can definitely remove this line. 👍

Reply:
Weird... I have the correct pylint version and `pre-commit` still complains about this. Can we just leave the pylint disable message? XD